README.md

```diff
@@ -134,6 +191,17 @@ Checkbox ``Don't throw Error on an Empty Result`` allows to emit an empty respon
 ![image](https://user-images.githubusercontent.com/40201204/43644579-f593d1c8-9737-11e8-9b97-ee9e575a19f7.png)
 As an input metadata you will get a Primary Key field to provide the data inside as a clause value.
 
+## Current limitations
+1. Only tables with one [PRIMARY KEY](https://en.wikipedia.org/wiki/Primary_key "PRIMARY KEY") is supported. Message ``Table has not Primary Key. Should be one Primary Key
+`` should be displayed if the table without any primary key will be selected and message ``Composite Primary Key is not supported
+`` should be displayed if the table with composite primary key will be selected.
+2. Only following versions of DataBase are supported:
+- ``MySQL`` - compatible with MySQL Server 5.5, 5.6, 5.7 and 8.0.
+- ``PostgreSQL`` - compatible with PostgreSQL 8.2 and higher
+- ``Oracle`` - compatible with Oracle Database 8.1.7 - 12.1.0.2
+- ``MSSQL`` - compatible with Microsoft SQL Server 2008 R2 and higher
+3. For Action "Upsert by primary key" metadata analysis ignores nullable/non-nullable fields, it will be throw SQL Server exception ``Cannot insert the value NULL into...`` if non-nullable fields don't fill for Insert operation. 
```

Comment:
The current implementation of the action ``Upsert By Primary Key`` doesn't mark non-nullable fields as required fields at a dynamic metadata. In case of updating such fields with an empty value you will get SQL Exception ``Cannot insert the value NULL into...``. You should manually fill in all non-nullable fields with previous data, if you want to update part of columns in a row, even if data in that fields doesn't change.